docs/getting-started/first-traffic-split.md

```diff
@@ -23,16 +23,16 @@ Instead of `TARGET`="World," let's update the environment variable `TARGET` on o
     ```
 
 === "YAML"
-
     ``` bash
+    ---
     apiVersion: serving.knative.dev/v1
     kind: Service
     metadata:
       name: hello
     spec:
       template:
         metadata:
-          name: knative
+          name: hello-knative
```

Comment:
The YAML is not matching the `kn` CLI 

If you want the revision to be `world` and `knative` and the service name to be named `hello` then you have to make the CLI refer to it as `world` and `knative` and not `hello-world` or `hello-knative`

Reply:
yeah this is correct I double check

Reply:
I think this is already fixed?